python/src/aiconfig/editor/server/server.py

```diff
@@ -335,7 +335,8 @@ def run() -> FlaskResponse:
 
     # Allow user to modify their environment keys without reloading the server.
     # Execution time of `0.001s` is arbitrary, but should be small enough to not be noticeable.
-    dotenv.load_dotenv()
+    # Should override be set to True?
```

Comment:
Maybe update comment to state something like,
"Override if env_file_path is specified so that provided .env values take precedence over existing values"